[UIK-2426][bulk-textarea] new bulk textarea component (#1893)
semrush/intergalactic: website/docs/components/bulk-textarea/bulk-textarea-a11y.md

| ---------------------------------------------- | -------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `BulkTextarea.InputField > div`                | `textbox`                                                | Identifies a field that allows the input of free-form text.                                                                                                    |
|                                                | `aria-multiline=”true”`                                  | Indicates that a textbox accepts multiple lines of input.                                                                                                      |
|                                                | `aria-labelledby="IDREF"`                                | Refers to the element that contains the label of field.                                                                                                        |

> |                                                | `aria-describedby="IDREF"`                               | Gives the item an accessible description by referring to the `aria-errormessage`, the tooltip, or both, describing the primary message or purpose of the item. |

Reviewer: When there is an error in a field, it will get a tooltip with the error text, which is linked to the field via `aria-describedby`
<img width="711" alt="image" src="https://github.com/user-attachments/assets/5e6918c9-f9ed-4315-b669-959596784503" />

> |                                                | `aria-describedby="IDREF"`                               | Gives the item an accessible description by referring to the `aria-errormessage`, the tooltip, or both, describing the primary message or purpose of the item. |

Reviewer 2: ```suggestion
|                                                | `aria-describedby="IDREF"`                               | Gives the item an accessible description. In invalid state, refers to the error message. |
```
what do you think?

> | `BulkTextarea.InputField > div > p`            | `aria-invalid="true"`                                    | A row gets this property when it gets the `invalid` status.                                                                                                    |

Reviewer 2: so InputField.Row component exists only in design docs/figma? :(

> |                                                | `aria-errormessage="[Your message here]"`                | Identifies the element that provides an error message for the .                                                                                                |

Author: I changed it to data-errormessage, we don't use it for VO. So, let's just remove it from here

> |                                                | `aria-errormessage="[Your message here]"`                | Identifies the element that provides an error message for the .                                                                                                |

Reviewer: I wouldn't remove it from the table, would just change it to:

> |   | `data-errormessage="Your message here"`  | Adds an error message for a line with `invalid` state.  |
